curious/graph.py

```diff
@@ -139,12 +143,17 @@ def get_related_objects(instances, filters=None):
       queryset = queryset.extra(select={INPUT_ATTR_PREFIX: '%s.%s' % (table, pk_field)})
 
     # reverse FK from instance to related objects with FK to the instance
-    elif type(rel_obj_descriptor) == ReverseManyToOneDescriptor:
-      rel_field = rel_obj_descriptor.rel.field
+    elif type(rel_obj_descriptor) in (ReverseManyToOneDescriptor, ReverseOneToOneDescriptor):
+      if type(rel_obj_descriptor) == ReverseOneToOneDescriptor:
+        rel_obj = rel_obj_descriptor.related
+      else:
+        rel_obj = rel_obj_descriptor.rel
```

Comment:
Might read clearer as an if else tertiary assignment expression

Reply:
Also kinda a pain that the name is different in this confusing wat

Reply:
There are a ton of copy/paste blocks in this function that should just figure out how to pick out the related object. I agree it's annoying the descriptor has a different attribute -- that may actually be fixable if one is just there for compatibility. I'll swap to a ternary

Reply:
Ha no, they don't have a common method. I'm surprised about that, but this still is all in the `not meant for public consumption category of Django`